src/openforms/js/lang/formio/nl.json

```diff
@@ -48,6 +48,7 @@
   "Press to open ": "Klik om te openen ",
   "or": "of",
   "browse": "blader",
+  "uploadFile": "upload {{fileFieldLabel}}",
```

Comment:
I think the `browse` translation may now be obsolete? I'm also wondering if a text like `selecteer bestand voor {{ fileFieldLabel }}` wouldn't work better?

The SDK PR is merged, and e2e tests are now breaking because of that, so those need to be updated in this PR too.

Reply:
`browse` translation is not obsolete - it's still used by the preview in our own form builder.

Reply:
Like discussed, the `browse` translation is now again used as the content of the anchor. The `selecteer een '{{ fileFieldLabel }}' bestand` is now used as the aria-label